lookout/style/typos/datasets_preparation.py

```diff
@@ -108,3 +123,37 @@ def prepare_data(params: Optional[Mapping[str, Any]] = None) -> pandas.DataFrame
     prepared_data = filter_splits(flat_data, vocabulary_tokens)[[Columns.Frequency, Columns.Split,
                                                                  Columns.Token]]
     return prepared_data
+
+
+def tune_fasttext_model(data: pandas.DataFrame,
+                        params: Optional[Mapping[str, Any]] = None) -> None:
+    """
+    Tune fasttext model on given data.
+
+    :param data: Dataframe which contains columns Columns.Split and Columns.Frequency.
+    :param params: Parameters for training the model, supports:
+                   size: Number of identifiers to pick from the given data to train fasttext on.
+                   corrupt: True to make random artificial typos in some part of the training data.
```

Comment:
Value indicating whether to make random artificial typos in some part of the training data.

Also: "some part" - which part?